Integrations/integration-PhishAI.yml

```diff
@@ -107,7 +107,7 @@ script:
                     ContentsFormat: formats.json,
                     HumanReadable: md,
                     EntryContext: {
-                        'Url(obj.Data.Url && obj.Data.Url===val.Data.Url)' : {
+                        'URL(obj.Data.Url && obj.Data.Url===val.Data.Url)' : {
```

Comment:
@idovandijk `obj.Data.Url && obj.Data.Url===val.Data.Url` Please change to `URL` inside here as well